src/main/java/org/jboss/remoting3/remote/RemoteReadListener.java

```diff
@@ -173,7 +173,7 @@ public void handleEvent(final ConduitStreamSourceChannel channel) {
 
                             final RegisteredService registeredService = handler.getConnectionContext().getRegisteredService(serviceType);
                             if (registeredService == null) {
-                                refuseService(channelId, "Unknown service name");
+                                refuseService(channelId, "Unknown service name " + serviceType);
```

Comment:
@dmlloyd if you're okay with adding service type at htis message, I'm going to create a Jira for it and submit this change

Reply:
Yeah this is fine